src/Altinn.AccessManagement.Core/Helpers/DelegationHelper.cs

```diff
@@ -317,6 +319,32 @@ public static void SetRuleType(List<Rule> rulesList, int offeredByPartyId, List<
             }
         }
 
+        /// <summary>
+        /// Extracts the (assumed) party ID from the given 'who' string. 
+        /// </summary>
+        /// <param name="who">
+        /// Who, valid values are "my", an organization number, or a party ID (the letter P followed by 
```

Comment:
"my" at least I don't think we should support in the same way as in Altinn 2

Reply:
Agree and particularly for this method, we are not handling "my". Removed "my"